kahuna/public/js/upload/view.html

```diff
@@ -22,27 +22,14 @@
 
     <file-prompt class="section" ng:if="!ctrl.latestJob"></file-prompt>
 
-    <section class="section">
+    <section class="section" ng:if="!ctrl.latestJob">
```

Comment:
How do you change "mode" of the upload page to reveal the last 50 uploads, having just uploaded images?